contrib/terraform/openstack/README.md

```diff
@@ -317,7 +317,8 @@ k8s_nodes:
    node-name:
     az: string # Name of the AZ
     flavor: string # Flavor ID to use
-    floating_ip: bool # If floating IPs should be created or not
+    create_floating_ip: bool # If floating IPs should be created or not
+    reserved_floating_ip: string # (optional) Use existing floating IP
```

Comment:
Suggestion: For backwards compatibility, should we do

```suggestion
    floating_ip: bool # If floating IPs should be used
    reserved_floating_ip: string # (optional) Use existing floating IP, if not set or set to "" a new IP will be created
```

So that the first variable only sets if an FIP should be used or not. And then the `reserved_floating_ip` dictates if a new FIP should be created or if an existing one should be used